src/backend/src/routes/organizations.routes.ts

```diff
@@ -36,8 +36,10 @@ organizationRouter.post(
 
 organizationRouter.post(
   '/contacts/set',
-  body('contacts').isArray(),
-  nonEmptyString(body('contacts.*')),
+  body('userIds').isArray(),
```

Comment:
hrm I would instead do an array of contacts, where each element is a userId and a title instead of two arrays.